msg/actuator_controls.msg

```diff
@@ -1,5 +1,5 @@
 uint64 timestamp			# time since system start (microseconds)
-uint8 NUM_ACTUATOR_CONTROLS = 8
+uint8 NUM_ACTUATOR_CONTROLS = 9
```

Comment:
Alternatively we could just get rid of this.

NUM_ACTUATOR_CONTROL_GROUPS as well

Reply:
NUM_ACTUATOR_CONTROLS is currently used in pca9685 - is that even something we still maintain?